### ISSUE TITLE
RTMP broken when rotation enabled - YouTube complains about resolution

### ISSUE THREAD

**Issue by jamescochran** [mrlt8/docker-wyze-bridge#431]
```
Error
You need to change the video resolution. The current resolution is (65535x65535), which is not supported for this configuration
```

This is after I added "YOUTUBE_FRONT_DOORBELL" and set it to my YT stream ID. The stream is still working but YT complains. 

p.s. I know I've added a few issues today and want to say this project is amazing! Just found it today and am loving it. Thank you!

**Comment by mrlt8** [comment]
I wonder if we _need_ to use a standard resolution or if something is wonky with our ffmpeg commands? Seems to be broken on https://demo.flashphoner.com/admin/demo.html when rotation is enabled.

**Comment by mrlt8** [comment]
This should be fixed in v1.6.3. 
Looks like we were missing some stuff in the SDP.
